docker-compose.yml

```
@@ -1,26 +1,24 @@
 version: '3.8'
 
 networks:
-  k6:
   grafana:
   prometheus:
```

```suggestion
  k6:
```
We should do the opposite, having a unique k6 network.

---

https://github.com/grafana/xk6-output-prometheus-remote/pull/148/commits/7d7c2ea62c971cd2af74db5f69061f10fa96d2b9